Xamarin.Forms.Core/SwipeDirection.cs

```diff
@@ -9,4 +9,38 @@ public enum SwipeDirection
 		Up = 4,
 		Down = 8
 	}
+	
+	public static class SwipeDirectionHelper
```

Comment:
should this needs to be public ?

Reply:
Added `[EditorBrowsable(EditorBrowsableState.Never)]`